lib/git-fastclone.rb

```diff
@@ -356,6 +356,7 @@ def with_git_mirror(url)
             'fatal: missing blob object',
             'fatal: remote did not send all necessary objects',
             /fatal: packed object [a-z0-9]+ \(stored in .*?\) is corrupt/,
+            'fatal: pack has 1 unresolved delta',
             /fatal: pack has \d+ unresolved deltas/,
```

Comment:
Could use this regexp and just delete the s at the end and it would cover the singular and plural cases

Reply:
👍